### PR TITLE
preliminay display number bpm

### PR DESCRIPTION
**Pull request by Ebdiel**
probando123

### CODE REVIEW

File: math.s43 [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include "msp430.h"                     ; #define controlled include file
+
+MODULE setup
```
Comment (kenneth-rosario) [inline math.s43]: Change module name to math

File: drawing.s43 [hunk 2]
```diff
@@ -44,14 +45,37 @@ displayBPM:
       MOV.B   #10100000B, &0X0a28  ;Mh
       ret  
       
+;Objetivo: coje el r10 y le hace display en lugar 0,1,2 
+;Precondiciones:  
+;Postcondiciones:  
+;Autor: Kenneth Rosario
```
Comment (kenneth-rosario) [inline drawing.s43]: Put your name here

**Review by kenneth-rosario — CHANGES_REQUESTED** [review]

File: math.s43 (hunk 1 above)
Comment (Ebdiel) [reply math.s43]: done

File: drawing.s43 (hunk 2 above)
Comment (Ebdiel) [reply drawing.s43]: done

File: drawing.s43 [hunk 3]
```diff
@@ -15,6 +21,83 @@ displayRead:
...
+      ret
+      
+displayOutline:
+      MOV.W   #2, &LCDCMEMCTL
+      MOV.B   #00010000B, &0X0a31  ; []
+      MOV.B   #00010000B, &0X0a2d  ; []]
+      ret
+ 
+displayfirstSegment:
+      MOV.W   #2, &LCDCMEMCTL
```
Comment (kenneth-rosario) [inline drawing.s43]: I think we do not need to clear the screen because we want them to appear together.

File: drawing.s43 [hunk 4]
```diff
@@ -15,6 +21,76 @@ displayRead:
       MOV.B   #01010000B, &0X0a33  ;dL
       ret
 
+;Objetivo: Hace display la bateria 
+;Precondiciones:  ninguna
+;Postcondiciones:  ninguna
+;Autor: Ebdiel Roman Feliciano
+;Fecha: 15/Abril/2021 
+displaybattery:
+      MOV.W   #2, &LCDCMEMCTL
```
Comment (kenneth-rosario) [inline drawing.s43:30]: en esta tambien

**Review by kenneth-rosario — APPROVED** [review]

**Pull request merged**